### PR TITLE
Swagger apiview new

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
@ruowan Can you please remvoe changes in APIView and APIViewWeb from this PR?

File: tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerTokenSerializer.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@
 using APIView;
 using APIViewWeb;
```
Comment (praveenkuttappan) [inline tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerTokenSerializer.cs:11]: Why are we linking to APIView?

Comment (praveenkuttappan) [reply tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerTokenSerializer.cs:11]: And APIViewWeb?

**Review by praveenkuttappan — APPROVED** [review]

File: tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerTokenSerializer.cs (hunk 1 above)
Comment (ruowan) [reply tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerTokenSerializer.cs:11]: Using codefilekind enum from apiVIew. And will using local enum and remove APIView dll in a seprate PR.

**Pull request merged**